### DISCUSSION TITLE
sharing post with multi language feature.

### DISCUSSION THREAD

**Discussion by mahidsec** [cotes2020/jekyll-theme-chirpy#1889] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Does Chirpy have any option where I can add a post with two languages and users decide which language they should read ( yes I have to write in two languages)?

### Operations you have already tried

I have searched for it but couldn't find anything

**Comment by mahidsec** [comment]
Never mind! I find a solution around it.

**Reply by geongupark** [reply]
@mahidsec How to solve the problem?? :)
Could you share the solution?

**Reply by mahidsec** [reply]
Just write two different post with same title and add an extra category at the end like en for English bn for Bangla cn for China and so on so forth
